tensorflow/python/ops/image_ops_impl.py

```diff
@@ -2168,6 +2168,14 @@ def adjust_brightness(image, delta):
       flt_image = image
     else:
       flt_image = convert_image_dtype(image, dtypes.float32)
+    
+    flag = True
+    for i in ((x.numpy().ravel())/255):
```

Comment:
`.numpy()` is not available in graph mode. You will not be able to use it here. 

Iterating through individual elements of a tensor and comparing them is not allowed, and raises `TypeError: Using a 'tf.Tensor' as a Python 'bool' is not allowed.` You can only use functions provided by the TensorFlow API inside a graph scope.

Same goes for `delta`.